--- control	2019-02-23 19:09:58.281807888 +0100
+++ control.new	2019-02-23 19:09:31.041476910 +0100
@@ -11,13 +11,12 @@
 # The compiler dependencies are relevant for backporting.
 	, g++ (>= 4.9) <!cross> | clang (>= 3.7) <!cross>
 	, gcc (>= 4.9) <!cross> | clang (>= 3.7) <!cross>
-	, cdbs, debhelper (>=10), dpkg-dev (>= 1.17.11~), lsb-release
+	, cdbs, debhelper (>=9), dpkg-dev (>= 1.17.11~), lsb-release
 	, libcppunit-dev
 	, libcap2-dev [linux-any]
 	, libdb-dev
 	, libecap3-dev (>= 1.0.1-2)
 	, libexpat1-dev
-	, libgnutls28-dev (>= 3.5)
 	, libkrb5-dev, comerr-dev
 	, libldap2-dev
 	, libnetfilter-conntrack-dev [linux-any]
