--- build/squid3/squid3-3.5.15/debian/rules	2016-02-17 01:13:33.000000000 +0100
+++ build/squid3/squid3-3.5.15/debian/rules.new	2016-02-22 22:50:04.079470555 +0100
@@ -45,7 +45,10 @@
 		--with-pidfile=/var/run/squid.pid \
 		--with-filedescriptors=65536 \
 		--with-large-files \
-		--with-default-user=proxy
+		--with-default-user=proxy \
+		--with-openssl \
+		--enable-ssl \
+		--enable-ssl-crtd
 
 BUILDINFO := $(shell lsb_release -si 2>/dev/null)
 
