--- control	2021-02-07 20:58:30.000000000 +0000
+++ control.new	2021-02-27 07:07:06.471623269 +0000
@@ -11,7 +11,7 @@
 # The compiler dependencies are relevant for backporting.
 	, g++ (>= 4.9) <!cross> | clang (>= 3.7) <!cross>
 	, gcc (>= 4.9) <!cross> | clang (>= 3.7) <!cross>
-	, debhelper (>=12), dpkg-dev (>= 1.17.11~), lsb-release
+	, debhelper (>=11), dpkg-dev (>= 1.17.11~), lsb-release
 	, dh-apparmor
 	, libcppunit-dev
 	, libcap2-dev [linux-any]
